WcaOnRails/spec/features/competition_management_spec.rb

```diff
@@ -14,6 +14,22 @@
       click_button "Create Competition"
 
       expect(page).to have_text("Successfully created new competition!")
+
+      visit "/competitions/new"
+      click_button "Create Competition"
+
+      expect(page).to have_text("Name must end with a year")
+    end
+
+    scenario "User clones an existing competition" do
```

Comment:
Maybe add in the description that you are testing the validation part? (like add "with validation errors" at the end)